code/modules/mob/living/simple_mob/subtypes/mechanical/corrupt_maint_drone.dm

```diff
@@ -0,0 +1,54 @@
+/*
+	A corrupted maintenance drone, produced from what seems like a bad factory.
+	They also tend to dodge while in melee range.
+	Code "borrowed" from viscerator drones. <3
+*/
+
+/datum/category_item/catalogue/technology/drone/corrupt_maint_drone

```

Comment:
You're editing a Polaris file. Might want to include a //Vorestation Addition comment thing here. Or make a _vr.dm file with an identical name, plus the underscore.

Reply:
This is an entirely new file, though?

Reply:
Ah, yes, it is. Just needs to be renamed to be, like, corrupt_maint_drone_vr.dm like we got for the vore panel in vorepanel_vr.dm